manifest.json

```diff
@@ -6,15 +6,15 @@
   "maintainer": "Flywheel <support@flywheel.io>",
   "source": "https://github.com/flywheel-apps/curate-bids",
   "url": "http://bids.neuroimaging.io/",
-  "version": "1.0.0_0.9.1",
+  "version": "1.1.0_0.10.0_dev3",
   "custom": {
-      "gear-builder": {
-           "category": "converter",
-           "image": "flywheel/curate-bids:1.0.0_0.9.1"
-      },
-      "flywheel": {
-          "suite": "BIDS"
-      }
+    "gear-builder": {
+      "category": "converter",
+      "image": "flywheel/curate-bids:1.1.0_0.10.0_dev3"
```

Comment:
Should the "dev3" be removed for the merge?

Reply:
Yes and the version will need to be updated to match bids-client.